### PR TITLE
RT-554:Summary card title is too long

### PR DESCRIPTION
**Pull request by yashpmohod**
## Overview

Add ellipses on the Dataset Label, and layer name.
Also, Add a hover tooltip on the dataset label, layer name, and summary card title.

## How it was tested

Tested using run showcase app locally.

## Checklist

- [x] The title contains a short meaningful summary
- [x] I have added a link to this PR in the Jira issue
- [x] I have described how this was tested
- [ ] I have included screen shots for changes that affect the user interface
- [ ] I have updated unit tests
- [x] I have run unit tests locally
- [ ] I have updated documentation (including README)

### CODE REVIEW

**Review by semicolin — CHANGES_REQUESTED** [review]
I suggest tweaking the `max-width`s a bit so the sidenav keeps the same size when switching panels, and to fill a bit more empty space on the summary card. See suggestions below.

File: projects/ngx-charts-on-fhir/src/lib/analysis-cards/analysis-card/analysis-card.component.css [hunk 1]
```diff
@@ -7,4 +7,8 @@
 }
 .mat-card-title {
   font-size: 20px;
+  max-width: 250px;
```
[the remark this answers is not in the document]
Comment (yashpmohod) [reply projects/ngx-charts-on-fhir/src/lib/analysis-cards/analysis-card/analysis-card.component.css]: Okay, I will change max-width:250px to max-width:350px

File: projects/ngx-charts-on-fhir/src/lib/data-layer-list/dataset-list/dataset-list.component.css [hunk 2]
```diff
@@ -9,3 +9,8 @@ mat-panel-title {
 mat-checkbox {
   margin-right: 8px;
 }
+mat-title{
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: 370px;
```
[the remark this answers is not in the document]
Comment (yashpmohod) [reply projects/ngx-charts-on-fhir/src/lib/data-layer-list/dataset-list/dataset-list.component.css]: Okay, I will change max-width:370px to max-width:300px

File: projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.css [hunk 3]
```diff
@@ -21,3 +21,10 @@
 mat-checkbox {
   margin-right: 8px;
 }
+
+mat-title{
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: 471px;
```
[the remark this answers is not in the document]
Comment (yashpmohod) [reply projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.css]: Okay, I will change max-width:471px to max-width:450px

File: projects/ngx-charts-on-fhir/src/lib/data-layer-list/dataset-list/dataset-list.component.html [hunk 4]
```diff
@@ -10,7 +10,9 @@
             [title]="dataset.hidden ? 'Show this dataset on the chart' : 'Hide this dataset on the chart'"
             color="primary"
           ></mat-checkbox>
-          {{ dataset.label }}
+          <mat-title matTooltip={{dataset.label}}>
```
[the remark this answers is not in the document]
Comment (yashpmohod) [reply projects/ngx-charts-on-fhir/src/lib/data-layer-list/dataset-list/dataset-list.component.html]: Okay, I will change that.

**Review by semicolin — APPROVED** [review]

**Pull request merged**